change-notes/1.19/analysis-cpp.md

```diff
@@ -24,6 +27,12 @@
 | Suspicious call to memset | Fewer false positive results | Types involving decltype are now correctly compared. |
 | Suspicious add with sizeof | Fewer false positive results | Arithmetic with void pointers (where allowed) is now excluded from this query. |
 | Wrong type of arguments to formatting function | Fewer false positive results | False positive results involving typedefs have been removed.  Expected argument types are determined more accurately, especially for wide string and pointer types.  Custom (non-standard) formatting functions are also identified more accurately. |
+| Missing return statement | Fewer false positive results | Functions which make a non-returning function call are no longer expected to have a return statement after that call. |
```

Comment:
There are already two entries for 'Missing return statement' above. I suggest merging all three into one.

Reply:
> It was 'medium' in release 1.17 and 1.18 because it had false positives due to an extractor bug that was fixed in 1.18.

Do you have any idea what this was a reference to?

Reply:
Thanks.  In the end I think we're saying too many different things to combine all three notes into one, but I've combined the two 'Fewer false positive results' notes into one - so we have only two change notes for this query now.